extensions/spring-xd-extension-sqoop/src/main/java/org/springframework/xd/sqoop/SqoopRunner.java

```diff
@@ -50,7 +50,7 @@ public static void main(String[] args) {
 			throw new IllegalArgumentException("Missing arguments and/or configuration options for Sqoop");
 		}
 
-		String jarPath = com.cloudera.sqoop.util.Jars.getJarPathForClass(JobConf.class);
+		String jarPath = org.apache.sqoop.util.Jars.getJarPathForClass(JobConf.class);
```

Comment:
Looks right but how the hell this can work if this PR didn't change deps? We have both on a classpath?